canvas_cli/apps/plugin/plugin.py

```diff
@@ -304,6 +331,17 @@ def validate_manifest(
 
     try:
         manifest_json = json.loads(manifest.read_text())
+
+        if new_protocols := _get_protocols_with_new_cqm_properties(
```

Comment:
this is where we check if the cqm properties in the protocols have differed from the manifest, and update the manifest if so.

Reply:
Are we sure we want to overwrite the manifest file? This also may not work, since there are multiple containers in prod.

Reply:
I should have seen the `canvas_cli` directory 🤦 . Looks good.